plutus-tx/src/PlutusTx/Trace.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
?

Reply:
Ooops, that was a vestage from the old implementation.